releasenotes/notes/fix-deadlock-in-metadata-e4c72281d54ac090.yaml

```diff
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fix potential deadlock when querying sub-processes configuration. When other processes are in a failed state the
+    Agent HTTPS timeout would not be taken into account causing to wait forever. This would block the creation of the status
+    page and cause missing information in flares.
```

Comment:
```suggestion
    Fix potential deadlock when querying sub-processes configuration. When other processes are in a failed state, the
    Agent HTTPS timeout would not be taken into account, causing it to wait forever. This would block the creation of the status
    page and cause missing information in flares.
```